app/models/operatingsystem.rb

```diff
@@ -186,26 +187,30 @@ def available_loaders
   end
 
   # sets the prefix for the tfp files based on the os / arch combination
-  def pxe_prefix(arch)
-    "boot/#{self}-#{arch}".tr(" ","-")
+  def pxe_prefix(arch, host)
```

Comment:
I'd suggest the `host` option to be optional, when not provided no digest is generated. This will give plugin authors more time to adopt. Discovery should be fine tho, it does not call this method directly.

Reply:
I didn't find any project that actually uses this method, and even if someone does, it could return wrong result without the host.
For example if someone is using this method to fetch the file, but relies on Foreman to put the files into the proxy, the result would be wrong. (Foreman will use hash for filename, but the plugin will look for non-hashed file).

Reply:
Okay.